Assets/Scripts/Stabable.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using static UnityEditor.PlayerSettings;
+
+public class Stabable : MonoBehaviour
+{
+
+    [SerializeField]Vector3 dashDir; //direction of the dash
+    [SerializeField]float dashLength; //how far the player is launched
```

Comment:
make both of theses variables `public` so the movement abilites have access to them